sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/models/WebAppBase.java

```diff
@@ -203,6 +203,13 @@ public interface WebAppBase extends HasName, GroupableResource<AppServiceManager
     /** @return an Observable streaming all logs */
     Flux<String> streamAllLogsAsync();
 
+    /**
+     * Whether the web app can be accessed from public network.
+     *
+     * @return whether the web app can be accessed from public network.
+     */
+    String publicNetworkAccess();
```

Comment:
@XiaofeiCao Should we add a `PublicNetworkAccess` class to the module, to keep the consistency?

Reply:
Seems this is the only module that not having it.

I'd prefer add one. Just copy it from another module, update doc, remove line 3 `// Code generated by Microsoft (R) AutoRest Code Generator.`.

Reply:
Please add a `PublicNetworkAccess` class to the module.

Reply:
Yeah, I think we should.